libvmaf/src/model.h

```diff
@@ -61,6 +66,11 @@ typedef struct VmafModel {
             bool enabled;
             double value;
         } p0, p1, p2;
+        struct {
+            bool enabled;
+            Point *list;
```

Comment:
Could this be a static allocation instead? i.e.`Point list[3]`?

Reply:
Could be. But I was following your example of VmafModelFeature *feature. I thought it's better to maintain a consistent style.